action.yml

```
@@ -0,0 +1,15 @@
+name: 'NPM Publish'
+description: 'NPM Publish the release'
```

They lowercase the name in their branding: https://www.npmjs.com/about

```suggestion
name: 'Publish to npm'
description: 'Publish the release to npm'
```